src/Microsoft.AspNetCore.Blazor.Build/targets/Blazor.MonoRuntime.targets

```diff
@@ -0,0 +1,305 @@
+        <Type>BlazorRuntime</Type>
+        <IsStatic>true</IsStatic>
+      </BlazorItemSpec>
+    </ItemGroup>
+
+    <PropertyGroup Label="Intermediate output paths">
+
+      <!-- /obj/debug/netstandard2.0/blazor -->
+      <BlazorIntermediateOutputPath>$(ProjectDir)$(IntermediateOutputPath)$(BaseBlazorIntermediateOutputPath)</BlazorIntermediateOutputPath>
+
+      <!-- /obj/debug/netstandard2.0/blazor/linker/ -->
+      <BlazorIntermediateLinkerOutputPath>$(BlazorIntermediateOutputPath)linker/</BlazorIntermediateLinkerOutputPath>
+      
+      <!-- /obj/debug/netstandard2.0/blazor/linker.marker.txt -->
+      <BlazorIntermediateLinkerResultFilePath>$(BlazorIntermediateOutputPath)linker.marker.txt</BlazorIntermediateLinkerResultFilePath>
+
+      <!-- /obj/debug/netstandard2.0/blazor/Index.html -->
+      <BlazorIndexHtmlOutputPath>$(BlazorIntermediateOutputPath)$(BlazorIndexHtmlName)</BlazorIndexHtmlOutputPath>
+    
+      <!-- /obj/debug/netstandard2.0/blazor/Descriptor.xml -->
+      <GeneratedBlazorLinkerDescriptor>$(BlazorIntermediateOutputPath)Descriptor.xml</GeneratedBlazorLinkerDescriptor>
+
+      <!-- /obj/debug/netstandard2.0/blazor/Descriptor.xml -->
+      <BlazorResolvedAssembliesOutputPath>$(BlazorIntermediateOutputPath)ResolvedAssemblies.txt</BlazorResolvedAssembliesOutputPath>
+
+    </PropertyGroup>
+    
+    <ItemGroup Label="Different outputs used through the build">
+      <ProducedLinkerDescriptor Include="$(GeneratedBlazorLinkerDescriptor)" />
+      <LinkerMarkerFileOutput Include="$(BlazorIntermediateLinkerResultFilePath)" />
+      <BlazorIndexHtmlFileOutput Include="$(BlazorIndexHtmlOutputPath)" />
+      <BlazorResolvedAssembliesOutput Include="BlazorResolvedAssembliesOutputPath" />
+    
+  </ItemGroup>
+   
+    <MakeDir Directories="$(BlazorIntermediateOutputPath)" />    
+    
+  </Target>
+
+  <Target Name="ResolveBinaryOutput" DependsOnTargets="CollectLinkerOutputs;GetBaseClassLibrary" />
+
+  <Target Name="CollectLinkerDescriptors"
+          Inputs="@(IntermediateAssembly);@(BlazorLinkerDescriptor);@(ProducedLinkerDescriptor)"
+          Outputs="@(ProducedLinkerDescriptor)"
+          Condition="$(BlazorShouldRunMonoLinker) == 'true' and '@(BlazorLinkerDescriptor)' == ''">
+    
+    <ItemGroup>
+      <PrepareAssemblyLine Include="@(IntermediateAssembly->'%(FileName)')" />
+      <GeneratedDescriptorLine Include="&lt;linker&gt;" />
+      <GeneratedDescriptorLine Include="@(PrepareAssemblyLine->'&lt;assembly fullname=&quot;%(Identity)&quot; /&gt;')" />
+      <GeneratedDescriptorLine Include="&lt;/linker&gt;" />
+    </ItemGroup>
+
+    <WriteLinesToFile
+      Condition = "'@(BlazorLinkerDescriptor)' == ''"
+      Lines = "@(GeneratedDescriptorLine)"
+      File ="$(GeneratedBlazorLinkerDescriptor)" Overwrite="true" />
+    
+    <ItemGroup Condition = "'@(BlazorLinkerDescriptor)' == ''">
```

Comment:
IMO just embrace that condition attributes in MSBuild are going to look message and kill all the extra whitespace.

Reply:
Not sure what that means if you mean the whitespace between Condition and = it'salready removed